draft-ietf-webpush-protocol.xml

```diff
@@ -1136,6 +1144,12 @@ HEADERS      [stream 82] +END_STREAM
           This could force the user agent to serialize subscription requests to the push
           service.
         </t>
+        <t>
+         A user agent that monitors subscriptions for other clients might
+         require multiple subscription sets. Each subscription set can then be
+         managed independently, which might include being moved between user

```

Comment:
@hruellan - thanks for the clean pull request. There is still the concern from @costinm from #96:

_BTW - I am also interested in the 'move between UA' use case, but I don't think  there is enough implementation and data around it._ 

Would you take the discussion to the mailing list? The other option is to simply remove that scenario.